testkit/src/main/scala/org/bitcoins/testkit/rpc/CachedBitcoind.scala

```diff
@@ -39,6 +36,44 @@ trait CachedBitcoindFunded[T <: BitcoindRpcClient] extends CachedBitcoind[T] {
     */
   protected val isBitcoindUsed: AtomicBoolean = new AtomicBoolean(false)
 
+}
+
+/** A cached bitcoind that has zero blocks in its chainstate. This is useful for
+  * testing at certain times when we need to make sure bitcoind is ONLY initialized, no chain state.
+  */
+trait CachedBitcoindNoFunds[T <: BitcoindRpcClient] extends CachedBitcoind[T] {
```

Comment:
I began implementing these to try and make a reproducible test case with bitcoind, but after talking with ben i decided to just handle this at `WalletRoutes`. I'm going to keep these fixtures around in case we need them in the future.